trivia-app/client/src/components/StartMenu.js

```diff
@@ -0,0 +1,69 @@
+const StartMenu = (props) => {
+  const handleAmount = (e) => {
+    props.setAmount(e.target.value);
+  };
+
+  const handleCategory = (e) => {
+    props.setCategory(e.target.value);
+  };
+
+  const handleDifficulty = (e) => {
+    props.setDifficulty(e.target.value);
+  };
+
+  return (
+    <div className="StartMenuDiv">
+      <div className="GameOptionsHeaderDiv">
+        <h2 className="GameOptionsHeader">Configure your game settings</h2>
+      </div>
+      <div className="GameOptionsDiv">
+        <form>
+          <label htmlFor="amount">
+            Select number of questions
+            <br />
+          </label>
+          <select className="DropDownMenu" onChange={handleAmount}>
+            <option className="placeholder" value="" disabled selected>
+              Choose a value
+            </option>
+            <option value="10">10</option>
+            <option value="15">15</option>
+            <option value="20">20</option>
+            <option value="25">25</option>
```

Comment:
do these value options (for amount/category/difficulty) come from an api? if so, would it be possible to iterate through the api response rather than hardcoding here?